### PR TITLE
feat(subscription):  add hover information on the status icon

### PR DESCRIPTION
**Pull request by manojava-gk**
## Description

add hover information on the status icon of app and service subscription

## Why

for better user experience

## Issue

#1123

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/docs/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

### CODE REVIEW

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Review by lavanya-bmw — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,21 @@
...
+- **App and Service Admin board**
+  - Fix data disappearing issue on re-clicking active tab/filter in app/service admin board [#1100](https://github.com/eclipse-tractusx/portal-frontend/pull/1100)
+- **App & Service Subcription Managment**
+  - Search place holder text updated
+  - Show only available offer names at the top
+  - Implement Hover-Information on the Status Icon of serviceSubscription
+- **Admin Service Management**
+  - Resolved service title overlap and improved responsiveness on the admin service detail page [#1112](https://github.com/eclipse-tractusx/portal-frontend/pull/1112)
+
+## Unreleased 2.2.0-RC3
```
Comment (evegufy) [inline CHANGELOG.md:22]: Please fix the changelog, this is not correct

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/assets/locales/en/main.json [hunk 2]
```diff
@@ -1456,6 +1456,9 @@
       "pleaseEnterValidURL": "Please enter a valid URL",
       "error": "Error",
       "success": "Success",
+      "pending": "Subscription is Pending",
+      "active": "Subscription is Active",
+      "process": "Subscription is in Process",
```
Comment (evegufy) [inline src/assets/locales/en/main.json:1461]: Are you certain about in process? Never saw that in the app up to now.
```suggestion
      "progress": "Subscription is in Progress",
```

File: src/assets/locales/de/main.json [hunk 3]
```diff
@@ -1488,6 +1488,9 @@
       "pleaseEnterValidURL": "Please enter a valid URL",
       "error": "Error",
       "success": "Success",
+      "pending": "Das Abonnement steht noch aus",
+      "active": "Das Abonnement ist aktiv",
+      "process": "Subscription is in Process",
```
Comment (evegufy) [inline src/assets/locales/de/main.json:1493]:
```suggestion
      "process": "Das Abonnement ist in Bearbeitung",
```

**Review by evegufy — CHANGES_REQUESTED** [review]

File: src/assets/locales/en/main.json (hunk 2 above)
Comment (manojava-gk) [reply src/assets/locales/en/main.json:1461]: Yes, progress is the right interpretation

File: src/assets/locales/de/main.json (hunk 3 above)
Comment (manojava-gk) [reply src/assets/locales/de/main.json:1493]: will update it

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,8 @@
   - updated technical user details with user type and service endpoint fields. Also, updated copy icon color for better view and made copy icon available only when value is available [#1137](https://github.com/eclipse-tractusx/portal-frontend/pull/1137)
 - **Company Subscriptions**
   - update API query parameters and fix tab UI scroll [#1131](https://github.com/eclipse-tractusx/portal-frontend/pull/1131)
+- - **App & Service Subcription Managment**
+  - Search place holder text updated, Show only available offer names at the top and Implement Hover-Information on the Status Icon of serviceSubscription[#1130](https://github.com/eclipse-tractusx/portal-frontend/pull/1130)
```
Comment (evegufy) [inline CHANGELOG.md:12]:
```suggestion
- **App & Service Subscription Management**
  - updated search place holder text - showing only available offer names at the top - and implemented hover information on the status icon of service subscription [#1130](https://github.com/eclipse-tractusx/portal-frontend/pull/1130)
```

**Review by evegufy — CHANGES_REQUESTED** [review]

**Review by evegufy — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,8 @@
   - updated technical user details with user type and service endpoint fields. Also, updated copy icon color for better view and made copy icon available only when value is available [#1137](https://github.com/eclipse-tractusx/portal-frontend/pull/1137)
 - **Company Subscriptions**
   - update API query parameters and fix tab UI scroll [#1131](https://github.com/eclipse-tractusx/portal-frontend/pull/1131)
+- **App & Service Subcription Managment**
```
Comment (evegufy) [inline CHANGELOG.md:11]:
```suggestion
- **App & Service Subscription Management**
```

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by manojava-gk** [comment]
@evegufy Could you pls check dependency issue

**Review by evegufy — CHANGES_REQUESTED** [review]
Please update dependencies file

**Review by evegufy — APPROVED** [review]

**Pull request merged**